src/stream/src/executor/managed_state/aggregation/value.rs

```diff
@@ -102,15 +101,23 @@ impl<S: StateStore> ManagedValueState<S> {
     }
 
     /// Flush the internal state to a write batch.
-    pub fn flush(&mut self, write_batch: &mut WriteBatch<S>) -> Result<()> {
+    pub async fn flush<S: StateStore>(
+        &mut self,
+        _write_batch: &mut WriteBatch<S>,
+        state_table: &mut StateTable<S>,
+    ) -> Result<()> {
         // If the managed state is not dirty, the caller should not flush. But forcing a flush won't
         // cause incorrect result: it will only produce more I/O.
         debug_assert!(self.is_dirty());
 
-        let mut local = write_batch.prefixify(&self.keyspace);
+        // Persist value into relational table.
         let v = self.state.get_output()?;
         // TODO(Yuanxin): Implement value meta
```

Comment:
Can this TODO be removed?

Reply:
cc @xx01cyx

Reply:
Yes, you can remove this. All value metas are computed within cell based table now, so there's nothing to do with `ManagedValueState `.